bot.py

```diff
@@ -138,16 +148,33 @@ async def get_rank(ctx, name: str):
             }
         )
         await ctx.send(file=file, embed=create_embed(embed_data))
-        # pylint: disable=broad-except
+
+    # pylint: disable=broad-except
     except Exception as e_values:
-        print(e_values)
+        # 404 error means Data not found in API
+        if "404" in str(e_values):
+            error_title = f'Summoner "{name}" is not found'
+            error_description = f"Please check the summoner name agian \n \
+              \n __*NOTE*__:   **{get_rank.name}** command only accepts one summoner name.\
+              \n\n Please type  `rank --help`  to see how to use"
+        else:
+            error_title = "Error"
+            error_description = "Oops! Something went wrong.\n\nPlease try again!\
+              \n\n Please type  `rank --help`  to see how to use"
```

Comment:
Same here. `error_description` can be shorted like https://github.com/dhmoon91/discord/pull/27/files#r668373752